analyzer/src/main/java/org/jboss/gm/analyzer/alignment/AlignmentTask.java

```diff
@@ -246,6 +254,23 @@ public void perform() {
                 cache.addGAV(project, current);
             }
 
+            AppliedPlugin ap = project.getPluginManager().findPlugin(DOKKA);
+            if (ap != null) {
+                logger.debug("Plugin {} has been applied to {}", ap.getId(), project.getName());
+
+                @SuppressWarnings("rawtypes")
+                Plugin p = ((DefaultPluginManager) project.getPluginManager()).getPluginContainer().findPlugin(DOKKA);
+                String path = p.getClass().getProtectionDomain().getCodeSource().getLocation().getPath();
+                Matcher m = Pattern.compile(".*-([\\d.]+[a-zA-Z]*)\\.jar").matcher(path);
```

Comment:
While I think `[\\d.]+` matches, it would also match `"..."? Maybe something like `\d+(\.\d+)*`?

Reply:
I odn't think we could ever have a jar installed using `.....` though surely?

Reply:
Right, it probably won't fail in practice, just pointing out the `[.]+`.

Reply:
Using https://regex101.com having played with e.g. 
```
gradle-enterprise-conventions-gradle-plugin-0.0.9.jar
bad-plugin-0.1..hello.jar
publish-plugin-1.1.0.jar
gradle-enterprise-conventions-gradle-plugin-0.0.9.jar
dependency-management-plugin-1.0.7.RELEASE.jar
propdeps-plugin-0.0.9.RELEASE2.jar
os-maven-plugin-1.4.0.Final.jar
weird-1.1.REL_11.jar
maven-settings-plugin-0.5.jar
```
a minor tweak produces `.*-([\d.]+[\w_]*)\.jar` which catches all of them and I think is reasonable.